wiki/tests/testdata/models.py

```diff
@@ -8,3 +8,7 @@ class CustomUser(AbstractUser):
         some_field = models.IntegerField(default=0)
 except ImportError:
     pass
+
+
+class CustomGroup(models.Model):
```

Comment:
As far as I saw in the code, the Group model only needs a __str__ function for display and a ForeignKey to User for permissions, but they don't seem to get tested so this blank model should suffice until then.